histoqc/tests/test_utils.py

```diff
@@ -0,0 +1,34 @@
+import io, os, re, cv2
+import pandas as pd
+
+# the two images are equal if the histogram correlation coefficient is greater than or equal to 0.95.
+def compare_images(image_path1, image_path2):
+    # Load the two images
+    image1 = cv2.imread(image_path1)
+    image2 = cv2.imread(image_path2)
+    # Check if the images have the same dimensions
+    if image1.shape == image2.shape:
+        # Compare the two images for exact equality
+        difference = cv2.subtract(image1, image2)
+        b, g, r = cv2.split(difference)
```

Comment:
Some of the output images, such as "mask_use" are single channel, but this function seems to only work for 3-channel images.

Reply:
Would you happen to have any suggestions for this one? Thanks